Common/game/main_game_file.cpp

```diff
@@ -608,7 +608,7 @@ void UpgradeCharacters(GameSetupStruct &game, GameDataVersion data_ver)
         {
             if (chars[i].legacy_scrname[0] == 0)
                 continue;
-            memcpy(namelwr, chars[i].legacy_scrname, LEGACY_MAX_CHAR_NAME_LEN);
+            memcpy(namelwr, chars[i].legacy_scrname, LEGACY_MAX_SCRIPT_NAME_LEN);
```

Comment:
This is correct, but then namelwr should also be made an array of this size, for consistency. (It is declared few lines above.)

Reply:
Ah, thanks, didn't notice it!